tests/integration/test_distributed_metadata_catalog/test.py

```diff
@@ -0,0 +1,132 @@
+import json
+import pytest
+import time
+
+from helpers.cluster import ClickHouseCluster
+
+cluster = ClickHouseCluster(__file__)
+
+node1 = cluster.add_instance('node1',
```

Comment:
This is existing clickhouse cluster we will not support this moving forward and in our test, i don't think we need this. 

If we just like to have 3 instances, then create 3 standalone daisy instance (internally, it will form a cluster automatically).